scripts/prepare_gitbook_site.py

```
@@ -0,0 +1,288 @@
+    """
+    output_lines: list[str] = []
+
+    for line in path.read_text(encoding="utf-8").splitlines():
+        stripped = line.strip()
+        if stripped.startswith(INCLUDE_PREFIX) and stripped.endswith(INCLUDE_SUFFIX):
+            include_path = stripped[len(INCLUDE_PREFIX) : -len(INCLUDE_SUFFIX)].strip()
+            source_path = REPO_ROOT / include_path
+            if not source_path.exists():
+                raise FileNotFoundError(
+                    f"Missing include `{include_path}` referenced from {path.relative_to(REPO_ROOT)}"
+                )
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)